openclaw_hooks/nima-memory/migrate_to_ladybug.py

```
@@ -0,0 +1,387 @@
+def check_ladybug_available() -> bool:
+    """Check if LadybugDB is installed and available"""
+    try:
+        import ladybug
+        return True
+    except ImportError:
+        return False
+
+def get_sqlite_stats() -> Dict[str, Any]:
+    """Get statistics from SQLite database"""
+    if not SQLITE_DB.exists():
+        raise FileNotFoundError(f"SQLite database not found: {SQLITE_DB}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)